### PR TITLE
Danish joint fit

### PR DESCRIPTION
**Pull request by jfcrenshaw**
Danish can now jointly fit a pair of donuts, rather than fitting individually and then averaging Zernikes. I think this is likely more robust, e.g. I think it is more likely to properly distinguish astigmatism from DCR. Futhermore, it appears to be about the same speed as the fit-individually-and-average method. I have, therefore, made the joint-fit the new default.

### CODE REVIEW

File: python/lsst/ts/wep/estimation/danish.py [hunk 1]
```diff
@@ -248,7 +297,7 @@ def _estimateSingleZk(
                     dy,
                     fwhm,
                     zkFit,
-                    sky_level=backgroundStd,
+                    sky_level=backgroundStd**2,
```
Comment (jmeyers314) [inline python/lsst/ts/wep/estimation/danish.py:300]: Good catch!

File: python/lsst/ts/wep/estimation/danish.py [hunk 2]
```diff
@@ -283,6 +332,172 @@ def _estimateSingleZk(
...
+                "zkFit": zkFit.copy(),
+                "zkSum": zkSum.copy(),
+                "model": modelImages[0],
+                "GalSimFFTSizeError": galSimFFTSizeError,
+            }
+            hist[I2.defocalType.value] = {
+                "image": imgs[1].copy(),
+                "variance": skyLevels[1],
+                "nollIndices": nollIndices.copy(),
+                "zkStart": zkStartI1.copy(),
```
Comment (jmeyers314) [inline python/lsst/ts/wep/estimation/danish.py]: `zkStartI2` ?

**Review by jmeyers314 — APPROVED** [review]
LGTM!  One small comment.  I guess you need for #299 to merge and rebase before merging this one?

File: python/lsst/ts/wep/estimation/danish.py (hunk 2 above)
Comment (jfcrenshaw) [reply python/lsst/ts/wep/estimation/danish.py]: nice catch!

**Pull request merged**